scylladb/sphinx-scylladb-theme
# fix: set sphinx-autobuild version

This PR solves the error raised by the latest version of `sphinx-autobuild` when attempting to build docs without a `_build/dirhtml` folder. The solution involves pinning a previous version of the extension until the bug is fixed.

**Error:**

> FileNotFoundError: [Errno 2] No such file or directory: '_build'

More details: https://github.com/sphinx-doc/sphinx-autobuild/issues/177

## How to test

### How to test

1. Run `make clean`
2. Run `make preview`
3. Ensure the project builds without errors.
